jbehave-support-core/docs/Reporting.md

```diff
@@ -32,10 +32,15 @@ There are several extensions already prepared and ready to use:
  - `WsXmlReporterExtension` (copies SOAP requests/responses from [WebServiceSteps](Web-service.md))
  - `TestContextXmlReporterExtension` (copies contents of [TestContext](Test-context.md))
  - `ServerLogXmlReporterExtension` (copies server log(s) for each system with configured [SshTemplate](Ssh.md))
+ - `SqlXmlReporterExtension` (copies SQL statements/results from [SqlSteps](Sql-steps.md))
  - `FailScreenshotsReporterExtension` (prints out error screenshots from [Web testing](Web-testing.md) - if any were generated)
  - `StepScreenshotsReporterExtension` (prints out step (not error) screenshots from [Web testing](Web-testing.md) - if any were generated)
- - `SqlXmlReporterExtension` (copies SQL statements/results from [SqlSteps](Sql-steps.md))
-
+   - Frequency of screenshot taking can be controlled by property: 'web.screenshot.type'
+     - MANUAL: screenshots from manual step only
+     - WAIT: screenshots after every web wait
+     - STEP: screenshots after every web step
+     - DEBUG: screenshots after every web step and action
```

Comment:
I would probably clarify it even more with something like: `(after every webdriver call)`

Reply:
I would keep it this way as it's not taking screenshots after verification etc. Variant with every web driver call didn't work as taking screenshot is web driver call to.